www/style/codecheckerviewer.css

```diff
@@ -425,4 +425,35 @@ span[class*="severity-"]::before {
 
 .comment .customIcon:hover {
   color: #000;
-}
\ No newline at end of file
+}
+
+/* Review status */
+.review-comment {
+  margin-left: 5px;
+  cursor: pointer;
+  color: grey;
+}
+
+.review-comment::before {
```

Comment:
If there is a chance, perhaps this icon should have a `cursor: help;` style added to it. I think it requires a new style element, not in the `::before`.